torch/_inductor/codegen/cpp.py

```diff
@@ -1314,19 +1319,21 @@ def where(a, b, c):
     @staticmethod
     def sign(x):
         code = BracesBuffer()
-        # auto tmp5 = tmp4 < 0 ? -1 : 1;
+        """
+        auto left = decltype(tmp1)::blendv(decltype(tmp1)(0), decltype(tmp1)(1), decltype(tmp1)(0) < tmp1);
+        auto right = decltype(tmp1)::blendv(decltype(tmp1)(0), decltype(tmp1)(1), tmp1 < decltype(tmp1)(0));
+        return left - right;
+        """
```

Comment:
why adding this comment. The code looks simple enough to know what code it would generate...

Reply:
Removed.